src/Provider/GoogleMaps/Tests/GoogleMapsTest.php

```diff
@@ -537,6 +537,22 @@ public function testReverseWithSubLocalityLevels()
         $this->assertEquals(false, $result->isPartialMatch());
     }
 
+    public function testGeocodeWithPostalCodeSuffixComponent()
+    {
+        $provider = $this->getGoogleMapsProvider();
+        $results = $provider->geocodeQuery(GeocodeQuery::create('900 S Oak Park Ave, Oak Park, IL 60304'));
+
+        $this->assertInstanceOf(AddressCollection::class, $results);
+        $this->assertCount(1, $results);
+
+        /** @var GoogleAddress $result */
+        $result = $results->first();
+        $this->assertInstanceOf(Address::class, $result);
+        $this->assertEquals('900 S Oak Park Ave, Oak Park, IL 60304, USA', $result->getFormattedAddress());
+        $this->assertEquals('1936', $result->getPostalCodeSuffix());
```

Comment:
I would add this assertion to the other tests so we are sure everything also works fine when `postal_code_suffix` is not set.

Reply:
Good idea, will do.